lib/iris/tests/unit/fileformats/pp/test__create_field_data.py

```diff
@@ -64,7 +64,7 @@ def test_deferred_bytes(self):
         field = mock.Mock(core_data=core_data)
         data_shape = (100, 120)
         proxy = mock.Mock(dtype=np.dtype('f4'), shape=data_shape,
-                          spec=pp.PPDataProxy)
+                          spec=pp.PPDataProxy, ndim=len(data_shape))
```

Comment:
OK, but I'm really surprised by this development.
Hitherto it was sufficient for a wrapped object to have just a `.dtype` and a `.shape`.
As ndim is a trivial consequence of shape, I wonder if this is a bit of a mistake in dask, and should be queried ??

Reply:
@pp-mo From what I understand there has been a bit of a refactor for `dask` version `2.0.0`, so for whatever reason they have started to access the `ndim` attribute of `dask.array` like objects.

Perhaps raise your concerns with them in an issue, if you care, but I don't see this as a blocker to this PR. The horse has kinda bolted, as they have already cut `dask` version `2.0.0` and now `2.1.0`.

Reply:
FWIW, raised https://github.com/dask/dask/issues/5106

Reply:
@pp-mo I see that itch has been scratched 😜